cmd/execution/main.go

```diff
@@ -218,6 +218,7 @@ func main() {
 			return nil
 		}).
 		Component("GCP block data uploader", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			computation.SetUploaderEnabled(enableBlockDataUpload)
```

Comment:
might want to do that inside the conditional incase the bucket name is empty. It should also get set to true in the s3 case (line 253)